docs/src/manifolds/metric.md

```diff
@@ -1,7 +1,46 @@
 # Metric manifold
 
+A Riemannian manifold always consists of a [topological manifold](https://en.wikipedia.org/wiki/Topological_manifold) together with a smoothly varying metric $g$.
+
+However, often there is a implicitly assumed (daefault) metric, like the usual
+inner product on [`Euclidean`](@ref) space. This decorator takes this into
+account. Hence it is not necessary to acts as a (as transparent as possible)
+decorator, see [`is_decorator_manifold`](@ref) for details.
```

Comment:
I don't really understand this sentence ("Hence it (...) for details"). Could you rephrase it?